.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+name: Build
```

This workflow does more than just build the project. Perhaps it can be better named?
```suggestion
name: CI
```